server/src/main/java/org/opensearch/cluster/service/ClusterManagerTaskThrottler.java

```diff
@@ -185,6 +189,27 @@ public void onBeginSubmit(List<? extends TaskBatcher.BatchedTask> tasks) {
         });
     }
 
+    /**
+     * If throttling thresholds are set via static setting, it will update the threshold map.
+     * It may start throwing throttling exception to older nodes in cluster.
+     *
+     * On submission of first task, it validates if all nodes are of 2.5.0 or not and
```

Comment:
For every task submission request, it will validate if nodes version >= 2.5.0 and set the startThrottling flag. Once the startThrottling flag is set, it will not perform check for next set of tasks.

Reply:
Ack